Detail on changing primary storage scope added to adminguide and concepts (#395)
apache/cloudstack-documentation: source/adminguide/storage.rst

|adding-local-pool-via-ui.png|

Changing the Scope of the Primary Storage
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Scope of a Primary Storage can be changed from Zone-wide to Cluster-wide
and vice versa when the Primary Storage is in Disabled state.
An action button is displayed in UI for each Primary Storage in Disabled state.

|change-storage-pool-scope-via-ui.png|

Scope change from Cluster to Zone will connect the Primary Storage to all Hosts
of the zone running the same hypervisor as set on the storage pool.

|change-storage-pool-scope-to-zone.png|

Scope change from Zone to Cluster will disconnect the Primary Storage from all
Hosts that were previously connected to the Primary Storage and are not a part
of the specified Cluster. So, if there are running VMs on such hosts using this
Storage Pool, it cannot be disconnected from the hosts. In this case the Scope
change operation will error out. Such VMs need to be stopped or migrated first.

> This feature is tested and supported for the following Hypervisor-Protocol-

Reviewer: ```suggestion
This feature is tested and supported for the following Hypervisor and storage combinations
```

Author: Done

> Storage Provider combinations:

Reviewer: ```suggestion

```

> -  KVM - NFS - Default Primary

Reviewer: ```suggestion
-  KVM with NFS
```

Author: Done

> -  KVM - CEPH/RBD - Default Primary

Author: Done

> -  VMWare - NFS - Default Primary

Reviewer: ```suggestion
-  VMWare with NFS
```

Author: Done
